### ISSUE TITLE
[CoE Starter Kit - QUESTION] CoE inventory reads and products Inactivity feature do not work well together.

### ISSUE THREAD

**Issue by saptarsibiswas** [microsoft/coe-starter-kit#7574]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

Inactive Environments still shows Active. after investigating by PG, they suggested to check with CoE team. 
Attached the list of Developer environments. Please let me know how we can unblock customer. 
[Dev environments.xlsx](https://github.com/microsoft/coe-starter-kit/files/14123825/Dev.environments.xlsx)

Tenant id: e11cbe9c-f680-44b9-9d42-d705f740b888

### What solution are you experiencing the issue with?

Core

### What method are you using to get inventory and telemetry?

None

AB#2180

**Comment by Jenefer-Monroe** [comment]
Hello. What version of Coe Core are you on, and what inventory type are you using? Sync flow v Data Export / BYODL

**Comment by saptarsibiswas** [comment]
Hello, the problem is, customer reported this with Power Platform and product group investigated and confirmed the behavior is an expected and known issue from CoE side.

The list of environments is not getting disabled. Inactive Environments still shows as Active. While investigating the issue, PG identified that some flows within the environments are still alive and that is the reason the environments are still active. 

Update from Product Group:
- This is the flow that is causing activity keeping the environments alive.
- https://make.powerautomate.com/environments/c2b3c9fd-85f5-e4c7-b4b2-4e0cce6d754a/flows/034d8ef1-7c22-ef06-7fc1-e35340ee818b/details
- Any flows that are running in your environment will trigger activity even if they are automated/scheduled. So, if you would like to have your environment disabled and cleaned up due to inactivity you should turn off all flows in the environment.

Customer stated the below:
Inventory flows from CoE keeping environments alive. We have two instances of CoE, one test and one production with about 50+ inventory flows in each instance, all using the List Environments action with the Power Platform Admin connector. So here customer confirmed that the flows cannot be deleted but still they want to be deleted the in-active developer environments. (Attached in the ticket)

**Comment by Jenefer-Monroe** [comment]
oh I see. This is about managed environments. This is a long going conversation in the product team not something that I can assist with. Will mark as product issue and allow the PMs in the space to interact.

**Comment by cr0max** [comment]
This was unexpected to see that our ticket ended up here. This is not about managed environments, from our side, but about our developer environment. They are kept alive through CoEs inventory flows. 
We had the same issues with Teams environments being kept alive, but that was solved by MS.

**Comment by Jenefer-Monroe** [comment]
Sorry ME is not required but it is the issue I was referencing. The product team PMS and the CoE PM (Grant) are working together on this. There isn't anything I can do about it as its just product behavior to count all reads as "activity"

**Comment by Jenefer-Monroe** [comment]
Hi there. Grant is actively working with the product team on a solution here, and will be your CoE POC going forward.

**Comment by kashanico** [comment]
Ok thank you. 
@Grant-Archibald-MS looking forward to hearing more about this issue. Our developer environments are building up for Makers who have left the company as activity keeps registering on them from the CoE.

**Comment by kashanico** [comment]
@Grant-Archibald-MS Bump - wondering if you have an update regarding this. Thnx.

**Comment by pnoot7** [comment]
Also looking for an update on this. Inventory for Teams environments is important, but the fact that it counts as activity is a big problem. I've set one team to be excluded from inventory as a test, but I suspect inventory is the cause. We need the auto clean up to be functioning.

**Comment by pnoot7** [comment]
Are there any updates that can be provided on this? I have an open support ticket open, and they claim working with product group, but nothing changes.

Inventory from CoE causes activity to be triggered, so the inactivity clean up process for environments never triggers as it should. 

I can exclude environments from inventory in the CoE, but then that defeats the purpose of the CoE.

**Comment by AmarSaiRam** [comment]
Dear User,

We noticed that this issue has been pending for quite some time, and we have not received any recent updates from your end. In the absence of further communication, we are proceeding to close this ticket, assuming that your concerns have been addressed or resolved.
If you continue to experience any difficulties related to this matter, please feel free to open a new support ticket. This will help us ensure prompt and effective assistance, rather than reopening the new Issue.
Thank you for your understanding and cooperation.

Best Regards,
CoE Starter Kit Team
